eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -109,32 +109,33 @@ $packageProperties = Get-ChildItem -Recurse -Force "$configFileDir" `
 foreach ($packagePropFile in $packageProperties)
 {
     $packageMetadata = Get-Content $packagePropFile | ConvertFrom-Json
-    Write-Host "Processing $($packageMetadata.ArtifactName)"
+    $artifactOrModuleName = $packageMetadata.ArtifactName ?? $packageMetadata.ModuleName
```

Comment:
What is different between artifact name and module name? Why can we not just put the module name as the artifact name property instead?

Reply:
Go is not setting artifact name. They are setting module name instead